src/main/resources/defaults/displayViews.yml

```diff
@@ -1099,7 +1137,7 @@
           field: geographicFocus
         - name: has sub grant
           template: "defaults/displayViews/relationships/overview/hasSubGrant.html"
-          field: hasSubGrant
+          field: subGrant
```

Comment:
Renamed field, but why not rename `hasSubGrant.html` as well?

Reply:
The property name is actually subGrant, https://github.com/TAMULib/scholars-discovery/blob/tamu-sprint10-204/src/main/java/edu/tamu/scholars/middleware/discovery/model/Relationship.java#L110.

Somehow, naming convention was not followed. There are a few others that differ from the template names.